packages/app-store/office365video/api/add.ts

```diff
@@ -6,7 +6,7 @@ import { WEBAPP_URL } from "@calcom/lib/constants";
 import { encodeOAuthState } from "../../_utils/encodeOAuthState";
 import getAppKeysFromSlug from "../../_utils/getAppKeysFromSlug";
 
-const scopes = ["OnlineMeetings.ReadWrite"];
+const scopes = ["OnlineMeetings.ReadWrite", "offline_access"];
```

Comment:
The `offline_access` scope is needed to receive a refresh token along with the access token